history/characters/10000_orc.txt

```diff
@@ -334,7 +334,6 @@
 	591.11.1={
 		culture=scourge religion=death_god
 		trait = being_undead
-		remove_trait = creature_orc
 		trait = creature_lich
```

Comment:
Ner'zhul shouldn't have an orc trait at this point. That's not what we try to achieve.
To make the race change working, there's fix that looks like:
```suggestion
		effect = { add_trait_force_tooltip = creature_lich }
```
Have to notice it's not a unique issue. You may find more by looking for the `remove_trait = creature_` pattern in `history\characters` folder.